install/kubernetes/helm/istio-remote/values.yaml

```diff
@@ -28,7 +28,14 @@ global:
     accessLogFile: "/dev/stdout"
 
     # If set, newly injected sidecars will have core dumps enabled.
-    enableCoreDump: false
+    # Deprecated, use maxNumCoreDumps below.
+    enableCoreDump: true
+    # Maximum number of coredumps to store (must be string due to helm numerical value handling inconsistencies):
+    #  0: no coredumps generated or stored
+    #  1: always overwrite <core_path>/core.proxy with most recent core.
+    #  unlimited: store all cores with timestamp at <core_path>/core.<timestamp>
+    # Any other value is currently unsupported and results in same behavior as value 1.
+    maxNumCoreDumps: "1"
```

Comment:
If you must add this option ( and I don't see a good use case): pattern should be 

coreDumps:
  max: 1
  enabled: true
  ....

At least that's what we do for other config groups

Reply:
Setting it to anything more than 0 means enabling it. Let's nuke "enabled", I do not see the justification for keeping it.

Reply:
I think this is unnecessary. max:0 is very clear vs. max:unlimited and enable:false is less so.

Reply:
Regarding deprecation, if we treat it as an API I think we should deprecate before removing. 
I'm arguing that maxCores is enough to handle all cases we need and we do not need separate enable/disable.